### DISCUSSION TITLE
About changing F# language color on GitHub to be more aligned to the language icon

### DISCUSSION THREAD

**Discussion by 64J0** [dotnet/fsharp#16563] [Polls]
Hey people, in GitHub the F# language is configured to appear using a pink/purple color, like this:

![image](https://github.com/dotnet/fsharp/assets/50725287/2fd459ed-ce08-48c9-8037-e9a967607d93)

But, the language icon is blue:

![image](https://github.com/dotnet/fsharp/assets/50725287/95d08c7e-7143-4cef-9fba-939c62303e91)

In order to make it more consistent, I open a PR to change the color to be the dark blue in the icon: `#348cbb`. Please let me know what you think about this.

The PR link is: https://github.com/github-linguist/linguist/pull/6686

**Comment by smoothdeveloper** [comment]
One thing, current colour is, to me, more saturated, and match the dotnet colour and the color VS uses, which I like.

The suggested colour makes F# closer to python, while current makes it closer to Fortran on github.

I'd be for keeping it as it is, based on the saturation mainly, even if it is not present in the official logo, and based on it is easier to keep things as is as many people recognize dotnet colour and F# colour on github already.

**Comment by i-shm** [comment]
![image](https://github.com/dotnet/fsharp/assets/66583620/cbc13291-9aa1-43e8-9050-4f05d7ff0892)

**Comment by 64J0** [comment]
Closing discussion in favor of https://github.com/dotnet/fsharp/discussions/18880
